packages/hw-transport-node-hid-singleton/src/TransportNodeHid.ts

```diff
@@ -88,6 +105,19 @@ export default class TransportNodeHidSingleton extends TransportNodeHidNoEvents
     };
   };
 
+  /**
+   * convenience wrapper for auto-disconnect logic
+   */
+  static async autoDisconnect(): void {
```

Comment:
can we call this function just `disconnect()` ? some transport have it and it's the same idea.